src/drc/api/viewsets.py

```diff
@@ -196,6 +201,8 @@ def get_serializer_class(self):
         """
         if self.action in ['update', 'partial_update']:
             return EnkelvoudigInformatieObjectWithLockSerializer
+        elif self.action == 'create' and self.request.data.get('inhoud', '') == '':
```

Comment:
probably fails in schema generation because there's no dynamic requests. maybe always use the serializer that has the lock field, and mark that field as optional/possibly empty + describe the behaviour in the help text.